src/Behat/Behat/Gherkin/Cli/FilterController.php

```diff
@@ -64,7 +65,13 @@ public function configure(Command $command)
                 '--role', null, InputOption::VALUE_REQUIRED,
                 "Only executeCall the features with actor role matching" . PHP_EOL .
                 "a wildcard."
-            );
+            )
+            ->addOption(
+                '--narrative', null, InputOption::VALUE_REQUIRED,
+                "Only executeCall the features with actor description" . PHP_EOL .
```

Comment:
Is `Only executeCall the...` here intentional (I realise it's in the existing descriptions too)? On first read it looks odd, should it just be `Only execute the...`?

Reply:
I just copied and pasted without paying much attention to the text, but you are right that this reads weird, updated